### PR TITLE
Use single yum repo cache

### PR DESCRIPTION
**Pull request by samuelson**
This is part of a set of interconnected PRs related to reducing VM size

### CODE REVIEW

File: manifests/master/pdf_stack.pp [hunk 1]
```diff
@@ -28,13 +28,7 @@
     require             => Yumrepo['robert-gcj'],
   }
 
-  # Regression in 0.12.4, https://github.com/wkhtmltopdf/wkhtmltopdf/issues/3241
```
Comment (binford2k) [inline manifests/master/pdf_stack.pp]: This does not seem to be relevant to the yum stack

File: manifests/agent/postfix_ipv4.pp [hunk 2]
```diff
@@ -1,4 +1,7 @@
 class classroom::agent::postfix_ipv4 {
+  package {'postfix':
```
Comment (binford2k) [inline manifests/agent/postfix_ipv4.pp]: is this being installed so that the package doesn't need to be cached? Or should it be in a separate pr?

File: manifests/master/pdf_stack.pp (hunk 1 above)
Comment (samuelson) [inline manifests/master/pdf_stack.pp]: yes, that kept throwing an error so I left it in.  Should I pull it into another PR @binford2k

File: manifests/agent/postfix_ipv4.pp (hunk 2 above)
Comment (samuelson) [inline manifests/agent/postfix_ipv4.pp]: @binford2k postfix was installed by the Puppetfactory Dockerfile, but we only need it in the classroom, so I moved it here. It needs to be pre installed or the augeas resource fails.

File: manifests/master/pdf_stack.pp (hunk 1 above)
Comment (samuelson) [inline manifests/master/pdf_stack.pp]: Also, this is tangentially relevant because of the version of the package to cache

**Comment by binford2k** [comment]
we do need to be better at atomic PRs
